server/grammar/YmlLexer.g4

```diff
@@ -51,6 +51,8 @@ SWITCH: 'switch';
 SWITCH_EXPR: 'switchExpr' | 'switchExprExclusive';
 CASE: 'case';
 DEFAULT: 'default';
+ACTION: 'action';
```

Comment:
Any reason why `noDefault` made its way into the RNs but not `action`?

Reply:
Yes, `noDefault` was a missing feature; `action` is here to remove ambiguity.